### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ostromart**
Fix some bad yamls and add links to samples.

### CODE REVIEW

File: samples/trafficmgmt-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: install.istio.io/v1alpha2
```
Comment (richardwxn) [inline samples/trafficmgmt-namespace.yaml]: can we use traffiicmanagement instead of trafficmgmt

File: samples/policy-off.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+apiVersion: install.istio.io/v1alpha2
```
Comment (richardwxn) [inline samples/policy-off.yaml]: A general comment about the sample configuration files name, would it be more clear to rename them as {base-profile}-{customization part}, e.g: sds-policy-off

File: README.md [hunk 3]
```diff
@@ -144,7 +164,7 @@ mesh profile dump --set profile=minimal
 
 #### Select a specific configuration profile
 
-The simplest customization is to select a profile different to `default` e.g. `sds`. Create the following config file:
+The simplest customization is to select a profile different to `default` e.g. `sds`. See [samples/sds.yaml](samples/sds.yaml):
 
 ```yaml
```
Comment (richardwxn) [inline README.md]: Do we still need to keep the below part or just pointing to samples is enough

Comment (ostromart) [reply README.md]: I think we should keep the text if users just want to read it.

File: samples/policy-off.yaml (hunk 2 above)
Comment (ostromart) [reply samples/policy-off.yaml]: Done.

File: samples/trafficmgmt-namespace.yaml (hunk 1 above)
Comment (ostromart) [reply samples/trafficmgmt-namespace.yaml]: Done.

**Pull request merged**